### DISCUSSION TITLE
Store products for other product types (OBF, OPF and OPPF)

### DISCUSSION THREAD

**Discussion by john-gom** [openfoodfacts/openfoodfacts-query#156] [Ideas]

**Comment by teolemon** [comment]
@stephanegigandet @john-gom I'm seeing products already moved a while ago (days) to Open Beauty Facts in this Open Food Facts facet, is that normal ? 
https://world.openfoodfacts.org/category/open-beauty-facts

**Comment by john-gom** [comment]
off-query gets the data from the MongoDB off>products collection. If the product doesn't exists there then we assume the product has been deleted. I therefore think the problem may be that the products aren't being deleted from the Food MongoDB collection when they are moved. See also https://github.com/openfoodfacts/openfoodfacts-server/issues/11330
